Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -1148,23 +1148,73 @@ lemma isReal_infinitePlace : InfinitePlace.IsReal (infinitePlace) :=
 
 end Rat
 
+namespace NumberField
+
+open InfinitePlace Module
+
+section TotallyRealField
+
 /-
 
 ## Totally real number fields
 
 -/
 
-namespace NumberField
-
 /-- A number field `K` is totally real if all of its infinite places
 are real. In other words, the image of every ring homomorphism `K → ℂ`
 is a subset of `ℝ`. -/
-class IsTotallyReal (K : Type*) [Field K] [NumberField K] where
+@[mk_iff] class IsTotallyReal (K : Type*) [Field K] [NumberField K] where
   isReal : ∀ v : InfinitePlace K, v.IsReal
 
+variable {K : Type*} [Field K] [NumberField K]
+
+theorem nrComplexPlaces_eq_zero_iff :
+    nrComplexPlaces K = 0 ↔ IsTotallyReal K := by
+  classical
+  simp [Fintype.card_eq_zero_iff, isEmpty_subtype, isTotallyReal_iff]
```

Comment:
Maybe also add `IsTotallyReal.nrComplexPlaces_eq_zero`?

Reply:
Yes, I guess it make sense to add as a simp lemma (and the same for totally complex fields). Thanks!